NExtends/Primitives/DateTimes/DateTime.extensions.cs

```diff
@@ -77,14 +77,14 @@ public static DateTime LastOfMonth(this DateTime d)
 		public static DateTime NextOrCurrent(this DateTime d, DayOfWeek day) { return LookFor(d, day, 1); }
 		static DateTime LookFor(DateTime start, DayOfWeek day, int step)
 		{
-			if (step % 7 == 0) throw new Exception("Boucle infinie");
+			if (step % 7 == 0) throw new Exception("Infinite loop");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') 'System.Exception' should not be thrown by user code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS112)